src/ol/source/tileutfgrid.js

```diff
@@ -479,7 +480,7 @@ ol.source.TileUTFGrid.Tile_.prototype.onXHRError_ = function(event) {
 
 
 /**
- * Load not yet loaded URI.
+ * @inheritDoc
```

Comment:
You couldn't use `@override` here? In the [docs](https://github.com/google/closure-compiler/wiki/Annotating-JavaScript-for-the-Closure-Compiler#override) it says:

> If no other annotations are included, the method or property automatically inherits annotations from its superclass.